Gabija_Balionyte.py

```diff
@@ -0,0 +1,95 @@
+import wave
+import numpy as np
+import matplotlib.pyplot as plt
+import os
+import soundfile as sf
+from tkinter import filedialog
+ 
+def openFile():  #Failų nuskaitymas kompiuteryje iš bet kurio aplanko filtruojant tik wav tipo failus. 
+    file_path = filedialog.askopenfilename(filetypes=[('any file','*.wav')])
+    return file_path
+
+def normalizeDataValues(data):  #Visos reikšmės sunormuotos 
+    data = data / data.max()
+    return data
+
```

Comment:
The local variable name 'file_path' hides the variable defined in the outer scope, making it inaccessible and might confuse